consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java

```diff
@@ -94,7 +94,7 @@ public RoundChangeManager(
   public Optional<Collection<RoundChange>> appendRoundChangeMessage(final RoundChange msg) {
 
     if (!isMessageValid(msg)) {
-      LOG.info("RoundChange message was invalid.");
+      LOG.debug("RoundChange message was invalid.");
```

Comment:
Why is an invalid RoundChange debug, but an invalid Proposal an info? (see IbftBlockHeightManager) - I think I prefer info in both places (this is an exceptional behaviour and we want to see it even normally)

Reply:
done